src/textformat.c

```diff
@@ -56,6 +56,7 @@ internal_format(
     colnr_T	leader_len;
     int		no_leader = FALSE;
     int		do_comments = (flags & INSCHAR_DO_COM);
+    int		safe_tw = trim_to_int((vimlong_T)(8 * (long)textwidth));
```

Comment:
```suggestion
    int		safe_tw = trim_to_int(8 * (vimlong_T)textwidth);
```

Reply:
Yes of course, casting to long breaks on Windows 🤦